stories/index.jsx

```diff
@@ -5,17 +5,24 @@ import { action } from '@storybook/addon-actions';
 import { linkTo } from '@storybook/addon-links';
 
 import { muiTheme } from 'storybook-addon-material-ui';
-
+import appTheme from '../web/js/theme/muiTheme'
 import { Button, Welcome } from '@storybook/react/demo';
 
 import Shares from '../web/js/components/shares';
+import AppBar from '../web/js/components/appbar';
+import {CardWF, MediaCardWF} from '../web/js/components/card_wf';
 
-storiesOf('Welcome', module).add('to Storybook', () => <Welcome showApp={linkTo('Button')} />);
-
-storiesOf('Button', module)
-  .add('with text', () => <Button onClick={action('clicked')}>Hello Button</Button>)
-  .add('with some emoji', () => <Button onClick={action('clicked')}>😀 😎 👍 💯</Button>);
 
 storiesOf('Shares', module)
-  .addDecorator(muiTheme())
+  .addDecorator(muiTheme(appTheme))
```

Comment:
Move the decorator to the config, so that we don't have to add it for each individual story (see documentation on how to do that